cmd/kubelet/app/server.go

```diff
@@ -251,6 +255,12 @@ is checked every 20 seconds (also configurable with a flag).`,
 				klog.InfoS("unsupported configuration:KubeletCgroups is not within KubeReservedCgroup")
 			}
 
+			if kubeletFlags.FailCgroupV1 {
```

Comment:
also: isn't this check duplicated with https://github.com/kubernetes/kubernetes/pull/126031/files#diff-d2cf199db99b0d2513c403aa1ae8c56e5e7c9b98345189bb8af50be0c3d7fd9aR30 
I think I prefer the one down there

Reply:
Sorry pushed the branch before addressing that. Let me fix it. Thanks.